packages/xod-cli/README.md

```diff
@@ -40,27 +40,29 @@ For example, instead of `--username` you can declare variable `XOD_USERNAME`.
 | Flag         | Alias | Environment variable |
 |--------------|-------|----------------------|
 | --api        |       | XOD_API              |
+| --board      | -b    | XOD_BOARD            |
 | --debug      |       | XOD_DEBUG            |
-| --help       | -h    |                      |
-| --no-build   |       |                      |
 | --on-behalf  |       | XOD_ONBEHALF         |
 | --output     | -o    | XOD_OUTPUT           |
 | --output-dir | -o    | XOD_OUTPUT           |
 | --password   |       | XOD_PASSWORD         |
-| --quiet      | -q    |                      |
+| --port       | -p    | XOD_PORT             |
 | --username   |       | XOD_USERNAME         |
-| --version    | -V    |                      |
 | --workspace  | -w    | XOD_WORKSPACE        |
 
 
 # Commands
 <!-- commands -->
 * [`xodc autocomplete [SHELL]`](#xodc-autocomplete-shell)
+* [`xodc boards [options]`](#xodc-boards-options)
+* [`xodc compile [options] [entrypoint]`](#xodc-compile-options-entrypoint)
 * [`xodc help [COMMAND]`](#xodc-help-command)
+* [`xodc install:arch [fqbn]`](#xodc-installarch-fqbn)
```

Comment:
:thinking: can it be just `install`. Think of it as of a package manager: XOD lib, C++ lib, hardware package, does not matter much

Reply:
OK, after a talk, nevermind, leave `install:arch`